lib/rubocop/cop/rspec/message_spies.rb

```diff
@@ -36,7 +36,7 @@ class MessageSpies < Cop
         SUPPORTED_STYLES = %w(have_received receive).freeze
 
         def_node_matcher :message_expectation, %(
-          (send (send nil :expect (send nil $_)) :to ...)
+          (send (send nil :expect {({send const} nil $_) ({ivar lvar} $_)}) {:to :to_not :not_to} ...)
```

Comment:
Can this line be somehow made more readable? As I recall it, line breaks have no special meaning to the AST.

Perhaps it would be good to extract a few variables?